templates/nagios_plugin.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
Shouldn't this file be in `files/check_neutron_networks.py` ? Is it a template?

Reply:
I was inpirated by `charm-kubernetes-master/templates/nagios_plugin.py `, but you are right. If I don't create a template from it, I'll move it to the `files` folder.